admin/tools/docker/replication/deployment/ncsa/run.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# This file is part of {{ cookiecutter.package_name }}.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+set -e
+
+# Load parameters of the setup into the corresponding environment
+# variables
+
+. $(dirname $0)/env.sh
+
+TOOL="$1"
+if [ -z "${TOOL}" ]; then
+    (>&2 echo "usage: <tool> [<parameters>]")
+    exit 1
+fi
+shift
+PARAMETERS="$@"
```

Comment:
I don't think `$@` does anything different from `$*`here.

Reply:
Though, it may not make any difference in my code, the former one is safer because it preserves quotes around arguments. See:
https://stackoverflow.com/questions/9915610/the-difference-between-and

Reply:
I don't think it works like that in this particular context.